torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20161,6 +20161,7 @@ def __init__(
     PythonRefInfo(
         "_refs.stack",
         torch_opinfo_name="stack",
+        validate_view_consistency=False,
```

Comment:
This will need a comment -- why is this no longer view consistent?

Reply:
oh, this was a question i had. The code is implemented like this, and I'm sure the first branch is triggered.

https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/TensorShape.cpp#L2212

But...

<img width="576" alt="image" src="https://user-images.githubusercontent.com/6355099/172446555-6f34c0db-1169-4372-9abb-10dec34c153b.png">

So why is it not considered a view?

Reply:
I don't know -- that does seem odd. 

We should either investigate until we have an issue, or maybe stack shouldn't be returning a view in the short-circuit case?

Reply:
View is an autograd concept; if you do your view below autograd it won't be tracked

Reply:
So should we update the tests to somehow track view more analogously to eager, or should we just xfail and move on?

Reply:
If we can arrange for the ref to get executed under autograd the view bit should get unset

Reply:
@mruberry I would vote for filing an issue for fixing up our "is_view" tests, and keeping this here. We have similar problems with other refs with similar behaviors: https://github.com/pytorch/pytorch/pull/78080#discussion_r885104613

Reply:
Filed an issue here: https://github.com/pytorch/pytorch/issues/79089